apps/admin-gui/src/assets/i18n/en.json

```diff
@@ -1330,6 +1323,12 @@
             "SUBMIT_BUTTON": "Delete",
             "CANCEL_BUTTON": "Cancel"
           }
+        },
+        "ATTRIBUTES_LIST": {
+          "ID": "Id",
+          "NAME": "Name",
+          "VALUE": "Value",
+          "DESCRIPTION": "Description"
```

Comment:
There are some locations that you must change when you move the localizations (eg, user-settings-attributes.component.html uses `SHARED.COMPONENTS.ATTRIBUTES_LIST.FILTER`. Please, find all these locations and fix them.